playbooks/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -14,14 +14,24 @@
         msg: "Please provide these required variable: ctdb_network_public_interface_name"
       when: ctdb_network_public_interface_name is undefined
 
-- name: Process backend specific tasks
-  include_tasks: "{{ config.be.name }}/main.yml"
-
 - name: Add ctdb package
   yum:
     name: ctdb
     state: present
 
+- name: Create lock directory
+  file:
+    path: "{{ config.paths.ctdb.var_lib }}/lock"
+    state: directory
+
+- name: Process backend specific tasks
+  include_tasks: "{{ config.be.name }}/main.yml"
+  vars:
+    ctdb_lock_path: "{{ config.paths.ctdb.var_lib }}/lock"
```

Comment:
I hope its OK to have mounts under _/var/lib/ctdb_.

Reply:
I haven't seen any complaint from selinux and all tests worked fine. An alternative is to use a symlink, but this seemed to me more prone to trigger selinux issues.